src/types/api.ts

```diff
@@ -144,21 +144,13 @@ export type UserInfo = {
   display_name: string | null;
 };
 
-export type Annotation = ClientAnnotationData & {
+export type ServerAnnotationData = {
```

Comment:
At first I was gonna say `APIAnnotation` seemed like a better option for naming, but then I saw how this pairs with `ClientAnnotationData`...hmmmm... 🤔

Reply:
What do you think about `APIAnnotationData` or `ServiceAnnotationData`?